projects/spirals-chaos/index.md

```diff
@@ -0,0 +1,126 @@
+--- 
+authors: 
+  - "Paulina Mrksic"
+authors-url: 
+  - "https://github.com/paulinalinaa"
+layout: project
+title: "spirals chaos"
+type: project
+---
+
+![](splash.png)￼
+
+
+
+# First Algorithm
```

Comment:
Please start at `##` (h2) see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax